prusti-common/src/config.rs

```diff
@@ -379,6 +380,10 @@ pub fn optimizations() -> Optimizations {
     return opt;
 }
 
+pub fn enable_purification_optimization() -> bool {
```

Comment:
Should have a short comment explanation like the other functions.

Reply:
Added.